client/replays/replay-drop.tsx

```diff
@@ -0,0 +1,44 @@
+import { useCallback, useEffect } from 'react'
+import { useAppDispatch } from '../redux-hooks'
+import { openReplay } from './action-creators'
+
+export default function ReplayDrop() {
+  const dispatch = useAppDispatch()
+
+  const onDrop = useCallback(
+    (e: DragEvent) => {
+      e.preventDefault()
+      e.stopPropagation()
+
+      const [file] = e.dataTransfer?.files || []
+      if (file?.path.endsWith('.rep')) {
+        dispatch(openReplay(file.path))
+      }
+    },
+    [dispatch],
+  )
+
+  const onDragOver = useCallback((e: DragEvent) => {
+    e.preventDefault()
+    e.stopPropagation()
+
+    // NOTE: it is not possible to access files during `dragover` now,
```

Comment:
I'm not sure that this is really worth noting, people in that issue seem to me to be misunderstanding the spec and I very much doubt this behavior will ever change back (it is most certainly a security issue if Chromium allows any random page to read files dragged over them but not actually dropped)